src/radical/pilot/pmgr/launching/base.py

```diff
@@ -980,7 +980,7 @@ def _prepare_pilot(self, resource, rcfg, pilot, expand, tar_name):
         jd_dict.queue                 = queue
         jd_dict.candidate_hosts       = candidate_hosts
         jd_dict.file_transfer         = list()
-        jd_dict.environment           = dict()
+        jd_dict.environment           = {'RP_PILOT_SANDBOX' : pilot_sandbox}
```

Comment:
as with `RADICAL_SMT`, let's move it next to `RADICAL_BASE` setup (L991)

Reply:
thanks, done.